docs/src/main/docs/metrics/01_metrics.adoc

```diff
@@ -16,22 +16,25 @@
 
 ///////////////////////////////////////////////////////////////////////////////
 
-= Metrics and Health Support
-:description: Helidon Reactive WebServer metrics and health support
-:keywords: helidon, reactive, reactive streams, reactive java, reactive webserver, metrics
+= Metrics
+:description: Helidon metrics
+:keywords: helidon, metrics
 
 == Metrics Support
-Helidon WebServer includes support for MicroProfile Metrics. MicroProfile
-Metrics has a variety of metric types and two reporting formats that
-are exposed at the `/metrics` endpoint.
+Metrics support in Helidon SE provides:
+
+1. Endpoint `/metrics` - a configurable endpoint that exposes metrics information in JSON (as specified by
+    Microprofile Metrics specification), or plain text (as used by Prometheus metrics)
+2. Base set of metrics, available on `/metrics/base`, as specified by Microprofile Metrics specification
```

Comment:
2. A base set of metrics, available at `/metrics/base`, as specified by the MicroProfile Metrics specification.

Reply:
Consider adding a link to the MP metrics spec.